python/lsst/sims/catUtils/utils/ObservationMetaDataGenerator.py

```diff
@@ -43,8 +54,8 @@ def __init__(self, driver=None, host=None, port=None, database=None):
                     stored in sims_data/OpSimData/
         """
         if database is None:
```

Comment:
I agree. But I was not sure if this functionality was being used in multiple tests .... so I wanted to leave this for a different PR. I can add a ticket.